include/umap.h

```diff
@@ -34,15 +34,14 @@ void* umap( void*  addr,    /* See mmap(2) */
             int    fd,      /* See mmap(2) */
             off_t  offset   /* See mmap(2) */
         );
+int uunmap( void*  addr,    /* See mmap(2) */
```

Comment:
We might. Currently in libumaptest, uunmap is called and then close the files.
We could put close files into uunmap_mf. But nothing else is different from uunmap.